MarketPlace.Backend/MarketPlace/Microservices/Authorization Service/Services.Authorization.Application/UseCases/UseCasesHandlers/DeleteUserUseCase.cs

```diff
@@ -1,23 +1,25 @@
-﻿namespace AuthorizationService
+﻿using MediatR;
+
+namespace AuthorizationService
 {
-    public class DeleteUserUseCase : IDeleteUserUseCase
+    public class DeleteUserUseCase : IRequestHandler<DeleteUserRequest>
     {
         private readonly IUserRepository _userRepository;
 
         public DeleteUserUseCase(IUserRepository userRepository)
         {
             _userRepository = userRepository;
-        }
+        }  
 
-        public async Task Execute(Guid userId, CancellationToken cancellationToken)
+        public async Task Handle(DeleteUserRequest request, CancellationToken cancellationToken)
         {
-            if (userId.Equals(Guid.Empty))
+            if (request.Id.Equals(Guid.Empty))
```

Comment:
```suggestion
            if (request.Id.Equals(Guid.Empty))
            {
                throw new FluentValidation.ValidationException("User Id must not be empty!");
            }
```
select one style with or without `{}`